pkg/network/ovn_kubernetes.go

```diff
@@ -52,6 +52,17 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.U
 		svcpools += fmt.Sprintf("%s", net)
 	}
 	data.Data["OVN_service_cidr"] = svcpools
+	data.Data["OVN_namespace"] = "openshift-ovn-kubernetes"
+
+	// ovnkube config file
+	data.Data["K8S_cluster_subnets"] = ippools
+	data.Data["OVNKUBE_loglevel"] = 2
+	data.Data["OVNKUBE_nb_privkey"] = ""
```

Comment:
I think for now for all these certs, let's just ignore them completely. When we do get SSL support we can add the template variables in.

Reply:
ok

Reply:
Changed